huaweicloud/services/apig/resource_huaweicloud_apig_instance.go

```diff
@@ -139,7 +139,6 @@ func ResourceApigInstanceV2() *schema.Resource {
 			"bandwidth_size": {
 				Type:         schema.TypeInt,
 				Optional:     true,
-				Computed:     true,
 				ValidateFunc: validation.IntBetween(1, 2000),
```

Comment:
value `0` is also allowed, please update the validation and docs

Reply:
done.